### PR TITLE
feat(block/languageswitcher): adds language switcher ui

### PR DESCRIPTION
**Pull request by naveenvignesh5**

### CODE REVIEW

File: react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
 function LanguageSwitcher() {
   return (
-    <>
-      <p className={styles.line}>LanguageSwitcher goes here.</p>
-    </>
+    <div className={styles.container}>
+      <div className={styles.langListContainer}>
+        <div className={styles.langList}>
+          Google offered in:&nbsp;
+          <Link className={styles.langLink} title="हिन्दी" />
```
Comment (ramsunvtech) [inline react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js:14]: put this value outside component and iterate with map

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js (hunk 1 above)
Comment (naveenvignesh5) [reply react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js]: resolved comment. kindly check @ramsunvtech.

File: react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js [hunk 2]
```diff
@@ -1,11 +1,34 @@
 import React from 'react'
+
+import Link from '@/ui/Link'
+
+// styles
 import styles from './LanguageSwitcher.module.css'
 
 function LanguageSwitcher() {
+  const languages = [
```
Comment (ramsunvtech) [inline react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js:9]: outside component

**Review by ramsunvtech — CHANGES_REQUESTED** [review]

File: react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js (hunk 1 above)
Comment (naveenvignesh5) [reply react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js]: Done @ramsunvtech. Kindly check.

**Review by ramsunvtech — APPROVED** [review]

**Pull request merged**